controls_pyqt/gui.py

```diff
@@ -215,6 +216,21 @@ def __initialize_layout(self):
         # self.layout.setColumnMinimumWidth(3, 400)
         self.layout.setColumnMinimumWidth(4, 600)
         self.setLayout(self.layout)
+    
+     def __switch_camera_layout(self):
+        if self.current_camera_layoutt == 0:
+            self.camera_view2.hide()
+            self.current_camera_layout = 1
```

Comment:
consider removing the `return` statements and place the constant setting of `self.current_camera_layout` with a `self.current_camera_layout += 1` and `self.current_camera_layout %= CAMERA_COUNT` or something similar